content/python/concepts/errors/errors.md

```diff
@@ -100,3 +100,43 @@ File "script.py", line 1, in <module>
 ratio = 100 / 0
 ZeroDivisionError: division by zero
 ```
+
+## Custom Exceptions
+
+Custom exceptions can be used to define and handle application-specific errors. This can be done by creating a sub-class of the built-in `Exception` class. Custom exceptions are useful for providing descriptive user feedback for unexpected behaviour. They can also help developers in debugging or be used elsewhere in the program. 
```

Comment:
The last 2 sentences can be merged together for better readability.

```suggestion
Custom exceptions can be used to define and handle application-specific errors. This can be done by creating a sub-class of the built-in `Exception` class. 

Custom exceptions help provide clear feedback to users about unexpected behavior, aid in debugging, and can be used throughout the program. 
```